src/dropzone/DropzoneDirective.ts

```diff
@@ -62,6 +62,17 @@ export class DropzoneDirective implements ng.IDirective {
             uploadToS3(scope.$ctrl, file, formData);
           }
         });
+      },
+      'success': function(file: any) {
+        this.removeFile(file);
+      },
+      'queuecomplete': function() {
+        // refresh bucket
+        // todo: bit of a hack, using querySelector to click 'Refresh'
+        // button instead of directly calling loadContents function
+        // defined in BucketController
+        let query = '.s3c-button[ng-click="$ctrl.loadContents()"]';
```

Comment:
Why don't you pass a method to the directive from the controller it can use to trigger this instead?

Reply:
I've tried a few things but still unable to access the 'loadContents' function in the scope of the dropzone directive. I've left this work around for now, but if you have any suggestions on where to make those changes, I can certainly add that here.

Reply:
I'll take a look.

Reply:
Thanks for sending me a diff of a different approach for fixing this issue (by using expression binding '&' to bind the parent scope's "loadContents" function). This change has been applied to the dropzone directive.